desktop/apps/collect/templates/index.jade

```diff
@@ -37,6 +37,8 @@ block body
           //- rendered on the client
       .cf-pillboxes
         //- rendered on the client
+      .cf-keyword
```

Comment:
this is still in the dom even if the lab feature is not switched on but it doesn't interfere with layout afaict.